tera-templates/macros.html

```diff
@@ -0,0 +1,77 @@
+{# Constructs a title based on the given crate name and version #}
+{% macro doc_title(name, version) %}
+    {%- if name -%}
+        {{ name }} {{ version | default(value="") }} - Docs.rs
+    {%- else -%}
+        Docs.rs
+    {%- endif -%}
+{% endmacro doc_title %}
+
+{#
+    Makes the appropriate JS imports for highlighting
+    * `languages` An array of strings where each is a valid highlight.js language
+#}
+{% macro highlight_js(languages) %}
```

Comment:
Nice use of function parameters, I didn't know tera could do that :)

Reply:
Yah, the only downside is that you cannot supply arrays as default arguments, I would have liked `highlight_js(languages=["ini", "rust"])` much better